src/libbson/src/bson/bson-version-functions.c

```diff
@@ -14,11 +14,9 @@
  * limitations under the License.
  */
 
-
 #include <bson/bson-version-functions.h>
 
-#include <bson/bson-version.h>
-
+#include <bson/version.h>
```

Comment:
`bson-version.h` is also referenced in `check-files.py`. I expect `check-files.py` can be removed (appears unused, and I think is no longer needed now that tarballs are not generated in CMake).